### ISSUE TITLE
Error after renaming a CSS file in the file manager

### ISSUE THREAD

**Issue by lionel-m** [contao/core#8624]
After renaming a CSS file in the file manager (Edit file or folder), the error below was displayed. Nevertheless the change was executed correctly.

Contao v3.5.23
Local -> MAMP

```
Warning: file_get_contents(/Applications/MAMP/htdocs/***/files/***/css/base2.css): failed to open stream: No such file or directory in system/modules/core/library/Contao/File.php on line 632

#0 [internal function]: __error(2, 'file_get_conten...', '/Applications/M...', 632, Array)
#1 system/modules/core/library/Contao/File.php(632): file_get_contents('/Applications/M...')
#2 system/modules/core/classes/Versions.php(176): Contao\File->getContent()
#3 system/modules/core/drivers/DC_Folder.php(1301): Contao\Versions->create()
#4 system/modules/core/classes/Backend.php(650): Contao\DC_Folder->edit()
#5 system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('files')
#6 contao/main.php(20): Contao\BackendMain->run()
#7 {main}


Warning: Cannot modify header information - headers already sent by (output started at /Applications/MAMP/htdocs/***/system/helper/functions.php:65) in system/modules/core/library/Contao/System.php on line 574

#0 [internal function]: __error(2, 'Cannot modify h...', '/Applications/M...', 574, Array)
#1 system/modules/core/library/Contao/System.php(574): setcookie('BE_PAGE_OFFSET', '0', 0, '/***', '', false, false)
#2 system/modules/core/drivers/DC_Folder.php(1334): Contao\System::setCookie('BE_PAGE_OFFSET', 0, 0)
#3 system/modules/core/classes/Backend.php(650): Contao\DC_Folder->edit()
#4 system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('files')
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}

```

**Comment by leofeyer** [comment]
Interestingly this only happens upon the first renaming attempt. After that, I can rename the file without any problems. Can you confirm this?

**Comment by lionel-m** [comment]
> After that, I can rename the file without any problems.

Yes this is also the case for me

**Comment by leofeyer** [comment]
And now I cannot reproduce the issue anymore. 🤔

**Comment by leofeyer** [comment]
Fixed in d171a9f676030ec2bef08f2b61b80b8eb58bd462.
